src/time.rs

```diff
@@ -488,3 +523,22 @@ impl PureTime {
         self.hour as u32 * 3_600 + self.minute as u32 * 60 + self.second as u32
     }
 }
+
+#[derive(Debug, Clone)]
+pub enum SecondsPrecisionOverflowBehavior {
+    Truncate,
+    Error,
+}
+
+#[derive(Debug, Clone)]
+pub struct TimeConfig {
+    pub seconds_precision_overflow_behavior: SecondsPrecisionOverflowBehavior,
+}
+
+impl Default for TimeConfig {
```

Comment:
better to do `derive(..., Default)` so you can do `SpeedateConfig {x: y, ...}`

Reply:
done